### PR TITLE
Drop-down added to registration form

### PR DESCRIPTION
**Pull request by shubhwale**

### CODE REVIEW

File: src/app/user/registration/registration.component.ts [hunk 1]
```diff
@@ -10,9 +11,23 @@ import { UserComponent } from '../user.component';
...
 
+  //array of data
+  myCities: {};
+  myStates: {};
+  gender: string[] = ['Male','Female','Others'];
+
+  
   ngOnInit(): void {
+    this.service.getCities().subscribe(cityList =>{
+      //console.log(cityList);
```
Comment (bala0105) [inline src/app/user/registration/registration.component.ts:24]: remove cosole.log();

File: src/app/user/registration/registration.component.ts [hunk 2]
```diff
@@ -10,9 +11,23 @@ import { UserComponent } from '../user.component';
...
+      this.myCities = cityList;
+    });
+    this.service.getStates().subscribe(stateList =>{
+      //console.log(stateList);
```
Comment (bala0105) [inline src/app/user/registration/registration.component.ts:28]: remove cosole.log();

File: src/app/user/registration/registration.component.ts (hunk 1 above)
Comment (shubhwale) [reply src/app/user/registration/registration.component.ts]: done

File: src/app/user/registration/registration.component.ts (hunk 2 above)
Comment (shubhwale) [reply src/app/user/registration/registration.component.ts]: done

**Pull request merged**